virtual_machine/windows_private/main.tf

```diff
@@ -61,3 +61,29 @@ resource "azurerm_virtual_machine" "virtual_machine" {
     builtby = "Terraform"
   }
 }
+
+resource "azurerm_metric_alertrule" "metric_alertrule" {
+  count               = "${var.number_servers}"
+  name                = "${var.application_name}_${var.hostname}_vm_${count.index + var.start_index}-cpu"
+  resource_group_name = "${var.resource_group}"
+  location            = "${var.location}"
+
+  description = "An alert rule to watch the metric Percentage CPU"
+
+  enabled = true
+
+  resource_id = "${element(azurerm_virtual_machine.virtual_machine.*.id, count.index + var.start_index)}"
+  metric_name = "Percentage CPU"
+  operator    = "GreaterThan"
+  threshold   = 90
+  aggregation = "Average"
+  period      = "PT5M"
+
+  email_action {
+    send_to_service_owners = false
+
+    custom_emails = [
+      "support@themisds.com",
```

Comment:
This is hard-coded whereas the others are variables, also, you should remove the e-mail address from this public shared repo before I merge this in.
Can you rebase over this to remove it from the git history before it gets re-submitted?